pyvips/vimage.py

```diff
@@ -324,11 +324,11 @@ def __exit__(self, type, value, traceback):
     def __getitem__(self, arg):
         if isinstance(arg, slice):
             i = 0
-            if arg.start != None:
+            if arg.start is not None:
```

Comment:
It's a style change, I received this from PEP8 (a style guide for Python code): 
`E711 comparison to None should be 'if cond is None:'`

See also: https://stackoverflow.com/a/3257957/1480019